Source/BoundaryConditions/WarpX_PEC.cpp

```diff
@@ -154,9 +148,9 @@ PEC::ApplyPECtoBfield (std::array<amrex::MultiFab*, 3> Bfield, const int lev,
         amrex::Array4<amrex::Real> const& Bz = Bfield[2]->array(mfi);
 
         // Extract tileboxes for which to loop
```

Comment:
Would you mind copying here the same comment that you added for the case of the E field above? I think it's a good explanatory comment, definitely worth having it here as well.

Reply:
Thank you for the suggestion. Added comment here. 6e884c7 
Also include the `const` suggestion in the .H and .cpp file for the ApplyPECtoEfield function argument.